index.js

```diff
@@ -484,17 +472,6 @@ RFID.prototype._wireReadData = function (numBytes, callback) {
   });
 };
 
-RFID.prototype._wireReadStatus = function () {
```

Comment:
This function is no longer used which is fantastic because `.read` seems to be the only inconsistency between T1 and T2 APIs. T1 returns `(value)` while T2 returns `(err, value)`.